spec/spec_helper.rb

```diff
@@ -16,6 +16,15 @@
     add_filter "spec/"
     add_filter "vendor/"
   end
+end if $LOAD_PATH.join.include?("simplecov")
```

Comment:
Perhaps something like this could work too:

```suggestion
end if Gem.loaded_specs.key?("simplecov")
```

Slight modification of https://stackoverflow.com/a/27532804

Reply:
Nice, thank you!